--- a/src/gnutls.c
+++ b/src/gnutls.c
@@ -350,7 +350,11 @@
     }
 
     /* 设置协议优先级 */
-    err = gnutls_protocol_set_priority(session, protocols);
+    err = gnutls_priority_set_direct(session, "NORMAL", NULL);
+    if (err < 0) {
+        logprintf (LOG_NOTQUIET, "GnuTLS: failed to set priority string: %s\n",
+                   gnutls_strerror(err));
+    }
 
     if (opt.debug)
         gnutls_session_set_ptr(session, (void *)"wget");
